### DISCUSSION TITLE
how to set dialog percentage width comparing to the main screen

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#15532] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I tried to set the percentage width of dialog comparing to the main screen, however it's not work.
https://codepen.io/Stanley-549393092/pen/eYLEaqG?editors=1010
Could you please have a look?

**Comment by MilosPaunovic** [comment]
Remove `position` attribute from your dialog.

**Reply by 549393092** [reply]
Yes, it could be a workaround. But I would like to use the position attribute. Thanks anyway!

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/vYzJqbL

**Reply by 549393092** [reply]
It's cool. Thanks a lot!
